api/web3server_marshal.go

```diff
@@ -29,8 +29,9 @@ type (
 	}
 
 	errMessage struct {
-		Code    int    `json:"code"`
-		Message string `json:"message"`
+		Code    int         `json:"code"`
+		Message string      `json:"message"`
+		Data    interface{} `json:"data,omitempty"`
```

Comment:
revert, do this in PR3920

Reply:
can not be reverted, expected API response needs to this `Data`